### DISCUSSION TITLE
Drawer example in docs

### DISCUSSION THREAD

**Discussion by ncoen-zaelot** [tailwindlabs/headlessui#2601] [Ideas]
Hey team!

I've just started using your library and I really like it. I noticed though, there's no example of a drawer component in the docs. Considering how common this element is, I reckon it could be a great fit for the Disclosure section.

Are you open to PRs for doc updates? If so, could you point me to the right place? If not, no worries, just wanted to throw this suggestion your way.

Thanks!

**Comment by segheysens** [comment]
@ncoen-zaelot can you clarify what do you need from a "Drawer" component that the current Disclosure doesn't do?

**Reply by ncoen-zaelot** [reply]
Sure! My suggestion is just about documentation. What I'm saying is that it would be helpful to have an already working example of a drawer (sidebar) because it's a really common case.
